docs/v2.0/develop/requests.md

```diff
@@ -94,13 +97,15 @@ All node specs begin with a node name: "expand-cluster", in this case. Node name
 
 makes Spin Cycle do `jobArgs["hostname"] = jobArgs["host"]` before passing `jobArgs` to the job.
 
-If expected == given, both must still be specified.
+Even if `expected == given`, both must still be specified.
 
 Only job args listed under `args:` are passed to the job. If a job needs arg "foo" but "foo" is not listed, then `jobArgs["foo"]` will be nil in the job. This requirement is strict and somewhat tedious, but it makes specs complete self-describing and easy to follow because there are no "hidden" args.
 
 If a job has optional args, they must be listed so they are passed to the job, in case they exist. The job is responsible for using the optional args or not. (Note: "optional" here is not the same as sequence-level optional args.)
 
-`sets:` specifies the job args that the job sets. The RM checks this. In the example above, the job sets "app", "env", and "node" in `jobArgs`. After calling the job's `Create` method, the RM checks that all three are set in `jobArgs` (with any value, including nil). Like `args:`, this is strict but makes it possible to follow every arg through different sequences. It also makes it explicit which jobs set which args.
+`sets:` specifies the job args that the job sets. The RM checks this. `arg:` and `as:` are to `sets:` as `expected:` and `given:` are to `args:` above: `arg:` is the job arg name set by the job, and `as:` is the job arg name in the specs to use. Even if `arg == as`, both must still be specified.
```

Comment:
My fault because I probably forgot to mention this :facepalm:, but can we make the `as` field optional? i.e. if you only put `args`, we assume `as` is the same.

Reply:
What's the reasoning behind making it optional here, but not for args/given/expected?

Reply:
We should have done it this way for expected/given too — it's a pain to write both all the time, and most of the time we don't rename args anyways. We should make `given` optional in the future as well.